### ISSUE TITLE
Add license information

### ISSUE THREAD

**Issue by Jsevillamol** [mlfoundations/model-soups#15]
Currently, this repository is provided without a license.
I recommend adding one to help clarify what users can (and not) do with the code.

**Comment by DongHwanJang** [comment]
I agree with the recommendation to add a license to this repo. Could you please provide any updates on this? Thanks :)

**Comment by mitchellnw** [comment]
done sorry
